File: 백준/Bronze/29340. Отряд/Отряд.py
import sys
input=lambda:sys.stdin.readline().rstrip()
sq1=[*map(int,input())]
sq2=[*map(int,input())]
r=""
for i in range(len(sq1)):
    r+=str(sq1[i]) if sq1[i]>sq2[i] else str(sq2[i])
print(r)

File: 백준/Bronze/29340. Отряд/Отряд.cc
#include <iostream>
#include <vector>
using namespace std;
int main(void) {
    string sq1, sq2, r = "";
    cin >> sq1;
    cin >> sq2;
    for (int i = 0; i < sq1.size(); i++)
        r += int(sq1[i]) > int(sq2[i]) ? sq1[i] : sq2[i];
    cout << r;
    return 0;
}